docs/resources/aviatrix_qos_policy_list.md

```diff
@@ -0,0 +1,56 @@
+---
+subcategory: "Copilot"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_qos_policy_list"
+description: |-
+  Creates Aviatrix QoS Policy List
+---
+
+# aviatrix_qos_policy_list
+
+The **aviatrix_qos_policy_list** resource creates the Aviatrix QoS Policy List.
+
+!> **WARNING:** Creating the **aviatrix_qos_policy_list** resource will overwrite all the QoS policies. Deleting the **aviatrix_qos_policy_list** resource will remove all the QoS policies.
+
+## Example Usage
+
+```hcl
+# Create a QoS Policy List
+resource "aviatrix_qos_policy_list" "test" {
+  policies {
+    name           = "qos_policy_1"
+    dscp_values    = ["1", "AF11"]
+    qos_class_uuid = "abcd1234"
+  }
+
+  policies {
+    name           = "qos_policy_2"
+    dscp_values    = ["AF22"]
+    qos_class_uuid = "efgh5678"
```

Comment:
i think you can make one of the examples here have a dependency on the QoS class resource `uuid` / `.id` . That way its clear that its not just a random string

Reply:
be sure to add an example QoS class resource within the snippet as well